### ISSUE TITLE
Decide whether Login.gov can meet need for identity proofing

### ISSUE THREAD

**Issue by cmajel** [USDAForestService/USFS-timber-permitting#76]
**Background**
Forest Service verifies the ID of folks purchasing firewood permits because they need to ensure that the permittee themselves is agreeing to abide by the rules of the permit. We want to understand whether Login.gov could be used to meet the need for permit purchasers to verify their identity and whether we could implement this capability under the existing Login.gov agreement. 

**Acceptance criteria** 
- [x] We understand whether Login.gov could be used to meet the requirement for permit purchasers to verify their identity and how it might impact the user experience. 

**Tasks**
- [x] Schedule mtg. w/ Laura Gerhardt and any others at Login. 
- [x] Draft Qs for the Login team
- [x] Conduct mtg. 

**Definition of done**
- [x]  Meets acceptance criteria

**Comment by MelissaBraxton** [comment]
Qs doc: https://docs.google.com/document/d/18F_Cv1mJqKaXdIKw9LdXR4s3EAP4Y6JAGt4DmSdH-Cc/edit?usp=sharing

**Comment by MelissaBraxton** [comment]
tl;dr - There are trade offs with going with Login.gov for ID verification. 

**CONS**
- The authentication flow is long, and we're worried folks will bounce before taking the time to go through it. 
 -  The new account creation flow is **very** long, and folks arriving to the permitting site without an account will have to set up an account before they can verify their identity. 
- A Login.gov account requires a social security number, and not every potential permit purchaser has one.

**PROS** 
- Already ATOd
- Open Forest already has an agreement with Login.gov, and a modification is currently in process--we could easily add ID verification onto to this mod to get started quickly with little  additional cost
- Login.gov is already integrated with the OF platform--USFS has already invested in it, and this will speed our path to production. 
- Likelihood that users landing on the site will already have Login account will increase as Login.gov continues to diffuse throughout the USDA and government more broadly
- A lot of permit purchasers are repeat customers--once they have an account and verified their ID one time, info is stored making subsequent visits quicker than they would be if we went with an alternative ID verification system that did not require account creation and therefore the verification process would need to be repeated every visit. 
 - Login.gov folks may be willing to work with us on tailoring the product and/or their service offerings to the unique needs of FS customers if needed.

**Comment by MelissaBraxton** [comment]
@bboddiger - Let's chat about these tradeoffs next week. My recommendation would be to pursue modifying the current agreement to add ID verification with Login.gov, but we'll want to understand the costs first. 

Follow up actions:

- [ ] Chat with Shadat and Aaron about timing of current mod and when we'd need to add on to it. 
- [ ] Estimate number of authentications and verifications we'd be adding (i.e., how many online firewood permits do we anticipate selling? in the pilot, after release 2, beyond...?)
 - [ ] Share this info with Login.gov folks to understand how much would be covered under the existing agreement and give us a cost estimate on any remainder.

**Comment by MelissaBraxton** [comment]
Comparative analysis sheet [here](https://github.com/USDAForestService/USFS-timber-permitting/issues/81).

**Comment by bboddiger** [comment]
Hi Melissa, thanks for this.  I read the comments you provided but haven’t been able yet to get to the documents themselves.  One question I do have is around password usage and whether that turns out to be an issue for folks who may only access once or twice a year for their annual permits.  Is a password even required for these accounts or do they just use ssn-do you know?

Beth Boddiger, USDA-FS National TIM (Timber Information Manager) Team Lead
WO Forest Management -Detached
2150 Centre Ave, BLDG A/STE 316
Fort Collins, CO  80526
Phone: 970-295-5792

**Comment by MelissaBraxton** [comment]
It is a great Q!

Once they have an account, the ID proofing feature verifies their ID using
their SSN.


Melissa Braxton
Experience Design, 18F
Cell: 425-770-2349
https://18f.gsa.gov/
